connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectClusterAssertions.java

```diff
@@ -44,9 +44,9 @@
 public class EmbeddedConnectClusterAssertions {
 
     private static final Logger log = LoggerFactory.getLogger(EmbeddedConnectClusterAssertions.class);
-    public static final long WORKER_SETUP_DURATION_MS = TimeUnit.SECONDS.toMillis(60);
+    public static final long WORKER_SETUP_DURATION_MS = TimeUnit.MINUTES.toMillis(5);
```

Comment:
This seems quite an increase. Have you tried with 2 minutes?

Reply:
I'm worried that with incremental increases, even if things are sufficient for now, they may become insufficient again in the near future. Given how infrequently changes that affect worker startup are (which might lead to this timeout elapsing before a true test failure is surfaced) and the number of moving parts that have to get initialized for an embedded Connect cluster to get off the ground, it seems worth the tradeoff to make the timeout high here.